src/tool/cppwinrt/test/inspectable_interop.cpp

```diff
@@ -0,0 +1,64 @@
+#include <inspectable.h>
+#include "winrt/Windows.Foundation.h"
+#include "catch.hpp"
+
+using namespace winrt;
+
+namespace
+{
+    struct __declspec(uuid("ed0dd761-c31e-4803-8cf9-22a2cb20ec47")) IBadInterop : ::IInspectable
```

Comment:
In practice, this IBadInterop definition would come from some ABI header file, right?

Reply:
Right.